components/style_derive/to_css.rs

```diff
@@ -18,7 +18,9 @@ pub fn derive(input: syn::DeriveInput) -> Tokens {
     let s = synstructure::Structure::new(&input);
 
     let match_body = s.each_variant(|variant| {
-        let bindings = variant.bindings();
+        let bindings = variant.bindings().into_iter().filter(|binding| {
+            !cg::parse_field_attrs::<CssFieldAttrs>(&binding.ast()).skip
```

Comment:
Would it make sense to just do `if attrs.skip { continue; }` in the loop? Unless we have a use case for `skip` + `iterable`, I don't think we should re-parse it again.

Reply:
I changed it this way.